client/src/routes/Publish/StepRegisterContent.module.scss

```diff
@@ -21,7 +21,7 @@
 
     a,
     button {
-        transition: color .2s ease-out;
+        transition: color 0.2s ease-out;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/manual/oceanprotocol/commons/pullRequest?prid=5548133)